src/Workspaces/Remote/Core/ProjectSystem/WorkspaceProjectCreationInfo.cs

```diff
@@ -12,4 +13,5 @@ internal sealed record WorkspaceProjectCreationInfo(
     [property: DataMember(Order = 0)] string Language,
     [property: DataMember(Order = 1)] string DisplayName,
     [property: DataMember(Order = 2)] string? FilePath,
-    [property: DataMember(Order = 3)] IReadOnlyDictionary<string, string> BuildSystemProperties);
+    [property: DataMember(Order = 3)] IReadOnlyDictionary<string, string> BuildSystemProperties,
+    [property: DataMember(Order = 4)] ImmutableArray<string> ProjectCapabilities = default);
```

Comment:
Is the expectation the project capabilities are going to be used for something else down the road, or is this only for logging?

Reply:
(maybe my real question is "the PR description tells me what this does, it doesn't tell me why")